modules/openapi-generator/src/main/resources/python/api.mustache

```diff
@@ -82,62 +82,63 @@ class {{classname}}(object):
                  returns the request thread.
         """
 
+        local_var_params = locals()
```

Comment:
IMO it's enough to start with catching local variables. Rest of the code don't use local variables directly, only via this `local_var_params`. Other created local variables (`collection_formats`) can't interfere with these `locals()`.

Reply:
For `collection_formats`, let's say if there's a required parameter named `collection_format` or `CollectionFormat`, I believe it will cause an issue with the local variable `collection_format` in the method, right?

Reply:
@tomplus thanks for performing some tests to confirm that 👍